packages/containerd/containerd-config-toml_basic

```diff
@@ -1,3 +1,4 @@
++++
```

Comment:
I believe we must always require the `+++` delimiter at the start of a template, otherwise it is impossible for the parser to appropriately disambiguate a template body with `+++` in it, especially if that body is a TOML document.

As far as who owns the templates, I've been thinking they may want to live in their own sub-packages that can be overridden, and use e.g. `Provides: template(kubelet-kubeconfig))`

Reply:
Re-reading this while not being in a rush, I agree that this probably shouldn't be a template at all.